ctrl_z/config.default.yml

```diff
@@ -31,7 +30,7 @@ files:
     - MEDIA_ROOT
 
 # Which binaries to use for backup creation/restore
-pg_dump_binary: /usr/bin/pg_dump
-pg_restore_binary: /usr/bin/pg_restore
-dropdb_binary: /usr/bin/dropdb
-createdb_binary: /usr/bin/createdb
+pg_dump_binary: /opt/homebrew/Cellar/libpq/16.0/bin/pg_dump
```

Comment:
This change makes sense for our working machines, but do we have Ctrl+Z running on any of our servers? Those would expect the linux paths, right?

Reply:
This is an example, there where we use CTRL-Z in our own servers we have a customised copy of this default.
I've changed it because otherwise our testcases would fail.